kube-client/src/api/portforward.rs

```diff
@@ -217,6 +228,12 @@ where
                     .await
                     .map_err(Error::ForwardFromPod)?;
             }
+            message if message.is_close() => {
+                sender
+                    .send(Message::FromPodClose)
+                    .await
+                    .map_err(Error::ForwardFromPod)?;
```

Comment:
I think we should stop the loop after receiving `close`.

```suggestion
                    .map_err(Error::ForwardFromPod)?;
                    break;
```

Reply:
Wouldn't that break if we forward more than one port in the same session?

Reply:
I wasn't sure if it was ok to do so tbh. 
On a separate issue, here and on the other `forwarder_loop` task any error means we basically error out. Now this means we drop any other ports, which might not be the right thing to do? anyway that's kind of a separate issue..

Reply:
There's only a single WebSocket connection even if we forward multiple ports.

Reply:
Ahh, I thought you meant we should stop after any one substream is closed. That should work, but I can also see an argument that both sides might as well try to close the connection once they know that it is useless.